src/ConfigAggregator.php

```diff
@@ -231,4 +256,39 @@ private function cacheConfig(array $config, $cachedConfigFile)
             var_export($config, true)
         ));
     }
+
+    /**
+     * @return array
+     */
+    private function postProcessConfig(array $processors, array $config)
+    {
+        foreach ($processors as $processor) {
+            $processor = $this->resolveProcessor($processor);
+            $config = $processor($config);
+        }
+
+        return $config;
+    }
+
+    /**
+     * @param Closure|object|callable $variable
+     *
+     * @return string
+     */
+    private function detectVariableType($variable)
+    {
+        if ($variable instanceof Closure) {
+            return 'Closure';
+        }
+
+        if (is_object($variable)) {
```

Comment:
I've switched the order for `is_object` and `is_callable` since `get_class` is imho more informational than `gettype` for callable objects.